bottlerocket-settings-models/CHANGELOG.md

```diff
@@ -9,7 +9,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - See [unreleased changes here]
 
-[unreleased changes here]: https://github.com/bottlerocket-os/bottlerocket-settings-sdk/compare/bottlerocket-settings-models-v0.4.0...HEAD
+[unreleased changes here]: https://github.com/bottlerocket-os/bottlerocket-settings-sdk/compare/bottlerocket-settings-models-v0.5.0...HEAD
+
+## [0.5.0] - 2024-09-10
+
+## Model Changes
+
+### Changed
+
+- Drop `nvidia-device-plugin` cargo feature ([#60])
```

Comment:
Can you also add an `### Added` line for the new settings extension? Even though it would refer to the same PR.